src/app/Upload.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+  let files = new Map<String, File>();
+
+  const handleDrop = (event: DragEvent) => {
+    console.log("File(s) dropped in the upload zone");
+    // Prevent default behavior (Prevent file from being opened)
+    event.preventDefault();
+
+    if (event.dataTransfer == null) {
+      return;
+    } else if (event.dataTransfer.items) {
+      // Use DataTransferItemList interface to access the file(s)
+      for (let i = 0; i < event.dataTransfer.items.length; i++) {
+        // If dropped items aren't files, reject them
+        if (event.dataTransfer.items[i].kind === "file") {
+          const file = event.dataTransfer.items[i].getAsFile();
+          if (file !== null) {
+            files.set(file.name, file);
+          }
+        }
+      }
+    } else {
+      // Use DataTransfer interface to access the file(s)
+      for (let i = 0; i < event.dataTransfer.files.length; i++) {
+        const file = event.dataTransfer.files[i];
+        alert(file);
+        files.set(file.name, file);
+      }
+    }
+    alert("File(s) dropped: " + files.size);
+  };
+
+  const handleDragOver = (event: DragEvent) => {
+    console.log("File(s) entered the upload zone");
+    // Prevent default behavior (Prevent file from being opened)
+    event.preventDefault();
+  };
+</script>
+
+<style>
+  #upload-zone {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 40%;
+    height: 400px;
+    background-color: rgb(97, 230, 186);
+  }
+
+  .inner-border {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    margin: 1em;
+    height: calc(100% - 2em);
+    width: calc(100% - 2em);
+    background-color: rgb(34, 153, 113);
+    border: 0.2em dashed rgb(17, 119, 85);
+  }
+
+  p {
+    color: white;
+  }
+</style>
```

Comment:
I would like to see some styles applied to upload zone when a file is currently being dragged (hovering) over the drop zone.